crates/context_aware_config/src/api/dimension/types.rs

```diff
@@ -8,6 +8,7 @@ pub struct CreateReq {
     pub schema: Value,
     #[serde(default, deserialize_with = "deserialize_option")]
     pub function_name: Option<Value>,
+    pub mandatory: Option<bool>,
```

Comment:
this should not be there right ?
users wont be able to change if a dimension is mandatory or not, right ?

as the data is coming from ENV at the moment

Reply:
forgot to remove